bcipy/signal/model/gaussian_mixture/gaussian_mixture.py

```diff
@@ -61,12 +59,16 @@ def predict(self, test_data: np.ndarray, means: np.ndarray, covs: np.ndarray):
 
         return likelihoods, predictions
 
-    def calculate_acc(self, test_data: np.ndarray, test_labels: Optional[np.ndarray]):
+    def calculate_acc(self, predictions: int, counter: int):
```

Comment:
Are these lists?

Reply:
These are integers. It was actually only used for plotting purposes (i.e. calculating the decision accuracy per symbol). It was a one liner accuracy calculation method that has nothing to do with the Gaussian Mixture class, so I will remove it.